assets/xml/overlays/ovl_Oceff_Spot.xml

```diff
@@ -0,0 +1,11 @@
+<Root>
+    <File Name="ovl_Oceff_Spot" BaseAddress="0x80BA6070" RangeStart="0x7F0" RangeEnd="0xEC8" Segment="128">
+            <Texture Name="sSunSongTex" OutName="sun_song_effect" Format="ia8" Width="32" Height="32" Offset="0x7F0"/>
+            
+            <Array Name="sSunSongCylinderVtx" Count="27" Offset="0xBF0">
+                <Vtx/>
+            </Array>
+            <DList Name="sSunSongMaterialDL" Offset="0xDA0"/>
+            <DList Name="sSunSongCylinderDL" Offset="0xE38"/>
```

Comment:
idk how they should be called but at the very least it should be consistent with similar data (other ovl_Oceff_*), like for song of storms currently:

```xml
        <DList Name="sSongOfStormsCylinderMaterialDL" Offset="0x1A08"/>
        <DList Name="sSongOfStormsCylinderModelDL" Offset="0x1AB0"/>
```
`Material`/`Cylinder` vs `CylinderMaterial`/`CylinderModel`